Get-Set-Struct-Properties.playground/Contents.swift

```diff
@@ -26,9 +44,11 @@ struct Student{
     }
 }
 
-var Ian = Student(firstName: "Ian", lastName:"Bansenauer", studentID: "920111123", _password: nil)
+var Sharief = Student(firstName: "Sharief", lastName:"Youssef", studentID: "920111123", _password: nil, streetNumber: "19128", streetName: "112th Ave NE", cityName: "Bothell", stateABV: "WA", zipCode: "98011")
 
-print("Initial password: " + Ian.password)
+print("Name: " + Sharief.fullName)
```

Comment:
You can also use the string literal notation in print, like
print("Name: (\Sharief.fullName)"